.gitignore

```diff
@@ -135,8 +135,6 @@ Tools/unicode/data/
 /config.log
 /config.status
 /config.status.lineno
-# hendrikmuhs/ccache-action@v1
-/.ccache
```

Comment:
I think we could leave this here, just remove the comment.  Even if there's nothing explicitly using it, it's never going to be something we want to check in.

Reply:
Re-added without comment.